src/__test__/operate.test.js

```diff
@@ -0,0 +1,64 @@
+import operate from '../logic/operate';
+
+test('expect 1 + 1 to be 2', () => {
+  // arrange
+  const operation = '+';
+  // act
+  const result = operate('1', '1', operation);
+  // assert
+  expect(result).toBe('2');
+});
+
+test('expect 2 - 1 to be 1', () => {
+  // arrange
```

Comment:
Hi @zhadier, these tests are well written, I love the effort ❤️ you have invested in this milestone 👏🏻. However, I can see that some tests are failing as seen below.

- [ ] Please kindly rerun them and find out why they are failing.

![image](https://user-images.githubusercontent.com/4247740/154376825-950480e5-3941-4a4e-b675-9ffd9a249edc.png)


- [ ] [OPTIONAL] It would also be nice if the tests where written under a describe block which helps to group tests testing similar of related functionality. For example
```
describe('Test operations', () => {
  test('Function returns a string', () => {
    expect(typeof (operate('2', '2', '+'))).toBe('string');
  });
```

- [ ] I will also mention that I really like the quote section of this app, - the shadow-color change to be exact 🥳

Reply:
Hi @Cathella  :wave:. Thank you so much for your help and guidance,  especially the optional suggestion because it's the optional suggestions that always teach you something new and that's what improves you as a developer. And Thank you for the compliment about the quote section, I used this tutorial for it. Hope it is useful :) https://www.youtube.com/watch?v=ZyNzmOEJ48A&t=1s